R/linter_tags.R

```diff
@@ -117,13 +117,10 @@ validate_linter_db <- function(available, package) {
   # Check that the csv file contains two character columns, named 'linter' and 'tags'.
   # Otherwise, fallback to an empty data frame.
   if (!all(c("linter", "tags") %in% colnames(available))) {
-    warning(
-      "`linters.csv` must contain the columns 'linter' and 'tags'.\nPackage '",
-      package, "' is missing ",
-      paste0("'", setdiff(c("linter", "tags"), names(available)), "'", collapse = " and "),
-      ".",
-      call. = FALSE
-    )
+    cli_warn(c(
+      i = "{.file linters.csv} must contain the columns {.str linter} and {.str tags}.",
```

Comment:
could this be a `{.field linter}` instead of `{.str linter}`?

Reply:
There is `{.val {c('linter', 'tags')}` if you want quoting + color

Reply:
I wanted the column names to be quoted, this is why I didn't use `.field` here.